doc/manipulating_images/data_preparation.rst

```diff
@@ -10,30 +10,53 @@ Data preparation: loading and basic signal extraction
 
 |
 
-.. topic:: **File names as arguments**
+.. currentmodule:: nilearn.image
 
-   Nilearn functions and objects accept file names as arguments::
+File names or image objects
+=============================
+
+File names and objects for 3D and 4D images
```

Comment:
Thanks a lot for the feedback. I have taken it in account. I am merging,
in the interest of short iterations (and because my time is up).